src/components/About.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Button, Flex, Heading, Paragraph } from "theme-ui";
+
+export const About = () => {
+  return (
+    <Box>
+      <Heading>About</Heading>
+      <Paragraph>
+        Hi there 👋, Welcome in Jam Assistant. A free open-source web app that
```

Comment:
```suggestion
        Welcome to Jam Assistant, a free open-source web app that
```

Reply:
Okay I will update the code and let you know